docs/6-rag/2-vector-stores.md

```diff
@@ -0,0 +1,171 @@
+# 📊 Vector Stores: The Foundation of RAG
+
+<div class="terminal-curl"></div>
+
+Think of vector stores as the specialized libraries that power RAG systems. While a traditional library organizes books alphabetically, vector stores organize information by **meaning**, allowing AI to find relevant content based on semantic similarity rather than just keyword matching.
+
+For educational platforms like Canopy, this means students can ask "How does bias affect hiring?" and get relevant information even if your course materials use terms like "discrimination in recruitment" or "algorithmic fairness in employment".
+
+## 🔍 What are Vector Stores?
+
+**Vector stores** (also called vector databases) are specialized databases that store and search through high-dimensional vectors - mathematical representations of text meaning. Here's how they work:
+
+
+![LLS RAG Architecture Diagram](images/rag5.png ':size=60%')
```

Comment:
Love this one!  
What do you think of also showing one where two different words get converted into two different (very small just to illustrate) embeddings?

Reply:
seems like a great idea! lemme add to the docs once I generated the image.

Reply:
done :)